tests/integrationtest/t/ddl/partition.test

```diff
@@ -135,3 +135,62 @@ show create table t;
 alter table t partition by key(a) partitions 5;
 show create table t;
 
+# TestPartitionInNonUTF8Charset
```

Comment:
Could you also add some `SHOW CREATE TABLE` output?

Reply:
Done :+1: